lib/ssl/src/ssl_logger.erl

```diff
@@ -290,19 +290,20 @@ get_server_version(Version, Extensions) ->
             Version
     end.
 
-version({3,4}) ->
+-spec version(ssl_record:ssl_version()) -> string().
+version(?'TLS-1.3') ->
     "TLS 1.3";
-version({3,3}) ->
+version(?'TLS-1.2') ->
     "TLS 1.2";
-version({3,2}) ->
+version(?'TLS-1.1') ->
     "TLS 1.1";
-version({3,1}) ->
+version(?'TLS-1.0') ->
     "TLS 1.0";
-version({3,0}) ->
+version(?'SSL-3.0') ->
```

Comment:
We should keep this one as we still allow this version for legacy client hellos!

Reply:
got it, so nothing to do :)